src/main/java/se/lucasboberg/Main.java

```diff
@@ -1,7 +1,22 @@
 package se.lucasboberg;
 
+import java.io.IOException;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        if (args.length < 2) {
+            System.out.println("No arguments given. You need to provide first a pattern and then a file path.");
+            return;
+        }
+        var pattern = args[0];
+        var path = args[1];
+
+        try {
+            var finder = new Finder(pattern, path);
+            finder.find();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Maybe you can handle this a little bit better. At least add logging.

Reply:
Fixed!